src/manifolds/ProbabilitySimplex.jl

```diff
@@ -281,6 +281,45 @@ mean(::ProbabilitySimplex, ::Any...)
 
 default_estimation_method(::ProbabilitySimplex, ::typeof(mean)) = GeodesicInterpolation()
 
+@doc raw"""
+    rand(::ProbabilitySimplex; vector_at=nothing, σ::Real=1.0)
+
+
+When `vector_at` is `nothing`, return a random (uniform) point `x` on the [`ProbabilitySimplex`](@ref)
+manifold `M` by normalizing a sample from an exponential distribution to unit sum.
```

Comment:
The reference refers to sampling from the simplex, not its tangent space:
```suggestion
manifold `M` by normalizing independent exponential draws to unit sum, see [^Devroye1986], Theorems 2.1 and 2.2 on p. 207 and 208, respectively.
```